x-pack/plugins/apm/public/components/app/TransactionDetails/Transaction/StickyTransactionProperties.tsx

```diff
@@ -91,5 +95,36 @@ export function StickyTransactionProperties({
     }
   ];
 
+  if (errorCount !== undefined) {
+    stickyProperties.push({
+      label: i18n.translate('xpack.apm.transactionDetails.errorsLabel', {
+        defaultMessage: 'Errors'
+      }),
+      val:
+        errorCount === 0 ? (
+          i18n.translate('xpack.apm.transactionDetails.errorsNone', {
+            defaultMessage: 'none'
+          })
+        ) : (
+          <KibanaLink
+            pathname={'/app/apm'}
+            hash={`/${idx(transaction, _ => _.service.name)}/errors`}
+            query={{
+              kuery: legacyEncodeURIComponent(
+                `transaction.id : "${transaction.transaction.id}"`
+              )
+            }}
+          >
+            {i18n.translate('xpack.apm.transactionDetails.errorsViewLink', {
```

Comment:
is there a process to request translations for these new texts?

Reply:
Not sure

Reply:
You can rename the label to `errorsOverviewLink`. That's the only way I know to invalidate the the contents.
But honestly: I don't get why we use separate keys in the first place, and not just the actual content.

Reply:
Asked around and apparently changing the content is enough.